src/components/Note/NoteReactionList/NoteReactionChip/NoteReactionChip.scss

```diff
@@ -0,0 +1,98 @@
+@import "src/constants/style";
+
+.note-reaction-chip__root {
+  all: unset; // remove default buttons styling
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  gap: 8px;
+
+  // total dimensions: 50x28
+  width: 35px;
+  height: 18px;
+  padding: 5px 8px 5px 7px;
+  border-radius: $rounded--full;
+  background-color: $color-lighter-gray;
+
+  font-size: $text--base;
+
+  user-select: none;
+  cursor: pointer;
+
+  &:hover {
+    background-color: $hover-background-light;
+  }
+}
+
+.note-reaction-chip__reaction {
+  font-size: $text--md;
+}
+
+.note-reaction-chip__amount {
+  font-size: $text--sm;
+  font-weight: 600;
+  line-height: 18px;
+  color: $color-black;
+}
+
+.note-reaction-chip__root--self {
+  background-color: var(--accent-color--desaturated-light);
+
+  &:hover {
+    background-color: var(--accent-color--desaturated-light-hover);
+  }
+}
+
+// override
+.note-reaction-chip__root--override-active {
+  background-color: var(--accent-color);
+
+  .note-reaction-chip__amount {
+    color: $color-white;
+  }
+
+  &:hover {
+    background-color: var(--accent-color); // essentially no hover color when active.
+  }
+}
+
+// 'div' is required (CSS specificity)
+div .note-reaction-chip__tooltip {
```

Comment:
For users with long names, the tooltip is cut off the the right side of the note. I'd either make it have a width smaller than the note and text-overflow: ellipsis or check if the tooltip could be rendered inside a portal so that it is shown outside of the note
<img width="418" alt="Bildschirmfoto 2023-09-28 um 11 08 30" src="https://github.com/inovex/scrumlr.io/assets/35272402/6ecca23f-4dac-40aa-b1ef-5948648e5c4d">

Reply:
if we keep the tooltip inside the note, then for a single reaction, using ellipsis works. But if multiple users reacted with the same emoji, names could get cut off completely. Could maybe be solved using multiline text, which also is the case for the `NoteReactionChipCondensed`, however text ellipsis isn't natively possible for multiline text (only at the end of the text but not per line).

Reply:
no overflow: name is getting cut off
<img width="325" alt="Bildschirm­foto 2023-10-12 um 11 49 50" src="https://github.com/inovex/scrumlr.io/assets/7889564/ff95e629-422e-44b5-9278-75947e28dafd">
overflow + ellipsis: no little arrow at the tooltip bottom, also cut off the third name
<img width="331" alt="Bildschirm­foto 2023-10-12 um 11 51 39" src="https://github.com/inovex/scrumlr.io/assets/7889564/2ff2a7d3-40c9-4d1a-af9e-76df97548f36">
condensed chip also isn't right
<img width="318" alt="Bildschirm­foto 2023-10-12 um 11 54 20" src="https://github.com/inovex/scrumlr.io/assets/7889564/380d5f0f-902c-4e86-a918-e19b3b257318">

Reply:
this is now fixed 👍